photon-api/src/main/scala/com/linkedin/photon/ml/util/MathUtils.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
```

Comment:
Yeah, there's no real need to update these, unless it's triggering OCD panic attacks.